dice.toml

```diff
@@ -0,0 +1,23 @@
+[Server]
```

Comment:
We already have .air.toml file, can we combine both of these?

Reply:
`air.toml` is supposed to be used by air ( live reloading tool ) and `dice.toml` is supposed to be our ( default ) config file from where it will read and loads the config value